### PR TITLE
renovate: automerge some dev stuff

### PR DESCRIPTION
**Pull request by fiws**
merge #54 first because it lints

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -1,5 +1,22 @@
 {
   "extends": [
     "config:base"
+  ],
+  "packageRules": [
+    {
+      "packagePatterns": ["eslint"],
```
Comment (guischdi) [inline renovate.json]: shouldn't there be ` "@hydrant/eslint-config", "eslint-plugin-ava"` be added here?

File: renovate.json [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+    },
+    {
+      "packageNames": [
+        "delay",
+        "ava",
+        "p-defer"
+      ],
+      "automerge": true,
+      "automergeType": "branch"
+    }
```
Comment (guischdi) [inline renovate.json]: what about `husky` and `lint-staged`?

File: renovate.json (hunk 1 above)
Comment (fiws) [reply renovate.json]: i think the pattern should match everything with eslint in the name

File: renovate.json (hunk 2 above)
Comment (fiws) [reply renovate.json]: those are not tested by the ci. should ideally be tested locally. or at least reviewed by hand

Comment (guischdi) [reply renovate.json]: alright, thanks for clarifying!

File: renovate.json (hunk 1 above)
Comment (guischdi) [reply renovate.json]: 👍

**Pull request merged**